src/irr/asset/CGLSLBRDFBuiltinIncludeLoader.h

```diff
@@ -342,6 +344,25 @@ float irr_glsl_beckmann(in float a2, in float NdotH2)
     return irr_glsl_RECIPROCAL_PI * nom/denom;
 }
 
+//TODO this is taking into account beckmann NDF only, i'd like to also account for masking/shadowing term (visibility of microfacets)
+BSDFSample irr_glsl_beckmann_cos_gen_sample(in uvec2 _sample, in float _a2)
```

Comment:
this should be `irr_glsl_beckmann_smith_height_correlated_cos_gen_sample`, I have no use for importance sampling just the NDF